supervisord.conf

```diff
@@ -3,33 +3,34 @@ nodaemon = true
 
 [program:nginx]
 command = /nginx.sh
+stopsignal = QUIT
 autostart = true
 autorestart = true
 stdout_logfile = /var/log/digitalmarketplace/nginx_access.log
 stdout_logfile_maxbytes = 50000000
 stdout_logfile_backups = 3
 stderr_logfile = /dev/stderr
 stderr_logfile_maxbytes = 0
-stopsignal = INT
 
 [program:uwsgi]
-command = uwsgi --ini /etc/uwsgi.conf --master
+command = uwsgi --ini /etc/uwsgi.conf --hook-master-start "unix_signal:1 gracefully_kill_them_all" --master
+# see https://github.com/unbit/uwsgi/issues/849#issuecomment-118869386 in regards to signal handling
+stopsignal = HUP
```

Comment:
If we're using HUP does that mean we no longer have the ability to reread config? Can we use the SIGINT signal number instead?

Reply:
https://uwsgi-docs.readthedocs.io/en/latest/Management.html#signals-for-controlling-uwsgi suggests that normally SIGTERM and SIGINT do the same thing, so mapping graceful shutdown to SIGINT doesn't lose us anything

Reply:
Sure.

Reply:
Done.